shap/plots/_decision.py

```diff
@@ -582,6 +583,7 @@ def multioutput_decision(base_values, shap_values, row_index, **kwargs) -> Union
         Returns a DecisionPlotResult object if `return_objects=True`. Returns `None` otherwise (the default).
 
     """
+    # todo: adjust to breaking changes made in #3318
```

Comment:
Do we need an issue to track this to-do? Is any functionality broken here at the moment?

Reply:
The functionality is not broken but pretty uncomfortable for the user but only in the case where we previously returned a list
So previously
```python
shap_values = explainer(X)
decision_plot(shap_values.base_values[0], shap_values)
```
worked but now one needs to construct these lists from the array (see the code I needed to test this: https://github.com/shap/shap/pull/3720/files#diff-a1e81e6553432a82ff71dd49ecdfeaf7a11d0af3a500605c303f15ab81fcee3bR95)